src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,50 @@
+import classNames from 'classnames';
+import { ChangeEvent } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { getSearchWith } from '../utils/searchHelper';
+import { SearchLink } from './SearchLink';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+  const sex = searchParams.get('sex') || null;
+
+  const allCenturies = ['16', '17', '18', '19', '20'];
+
+  function setSearchWith(params: any) {
```

Comment:
```suggestion
  function setSearchWith(params: SearchParams) {
```
import this type from react-router-dom
![image](https://github.com/user-attachments/assets/28bba035-ea17-4cbc-a438-15962f6d8d84)

FYI